Shop.DAL/Models/OrderItem.cs

```diff
@@ -0,0 +1,7 @@
+namespace Shop.DAL.Models; 
+
+public class OrderItem: BaseEntity{
+    public Guid GameId {get;set;}
+    public Guid OrderId {get;set;}
+    public int Count {get;set;}
```

Comment:
here it is also necessary to write the entities themselves, like public Order? Order {get; set;}